### PR TITLE
Participants table

### PR DESCRIPTION
**Pull request by BarbaMatei**

### CODE REVIEW

File: api/Data/DataContext.cs [hunk 1]
```diff
@@ -13,5 +13,14 @@ public DataContext(DbContextOptions options) : base(options)
     public DbSet<User> Users { get; set; }
     public DbSet<Event> Events { get; set; }
     public DbSet<Location> Location { get; set; }
+    public DbSet<Attendance> Attendance { get; set; }
+
+    protected override void OnModelCreating(ModelBuilder builder)
+        {
+
```
Comment (Adrian3110) [inline api/Data/DataContext.cs]: Format this file and remove this empty line

File: api/Service/AttendanceService.cs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+using System.Linq;
+
+namespace api.Service
+{
+	public class AttendanceService : IAttendanceService
+	{
+        private readonly DataContext _context;
+        private readonly IMapper _mapper;
+        public AttendanceService(DataContext context, IMapper mapper)
+		{
```
Comment (Adrian3110) [inline api/Service/AttendanceService.cs:17]: Format this file also

**Review by Adrian3110 — CHANGES_REQUESTED** [review]

File: api/Service/AttendanceService.cs (hunk 2 above)
Comment (Adrian3110) [inline api/Service/AttendanceService.cs:17]: Format this file also

File: api/DTOs/AttendanceDto.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using YonderfulApi.DTOs;
+
+namespace YonderfulApi.DTOs
+{
+    public class AttendanceDto
+    {
+        [Required]
+        public int EvenimentId { get; set; }
```
Comment (meda-racasan) [inline api/DTOs/AttendanceDto.cs]: Use English namings

File: api/Migrations/20220328124415_Attendance.cs [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.RenameColumn(
+                name: "Autocancel",
+                table: "Events",
+                newName: "AutoCancel");
+
+            migrationBuilder.CreateTable(
+                name: "Attendace",
```
Comment (meda-racasan) [inline api/Migrations/20220328124415_Attendance.cs:16]: You have a typo in the name of the table; should be Attendance

File: api/Models/Attendance.cs [hunk 5]
```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace YonderfulApi.Models
+{
+    public class Attendance
+    {
+        public int EvenimentId { get; set; }
```
Comment (meda-racasan) [inline api/Models/Attendance.cs]: Use English names

File: api/Service/IAttendanceService.cs [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+namespace YonderfulApi.Service
+{
+    public interface IAttendanceService
+    {
+        Task<Attendance> GetAttendance(int EventId, int UserId);
+        Task<IList<Attendance>> GetAllAttendance();
+        Task<IList<Attendance>> GetParticipants(int EventId);
+        Task<IList<Attendance>> GetEventsForUser(int UserId);
+        Task<Attendance> PostAttendance(Attendance newAttendance);
```
Comment (meda-racasan) [inline api/Service/IAttendanceService.cs]: I'd call these methods
 PostAttendance -> CreateAttendance
 PutAttendance -> UpdateAttendance

File: api/Service/IAttendanceService.cs [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+        Task<Attendance> PutAttendance(Attendance attendanceToPut);
```
Comment (meda-racasan) [inline api/Service/IAttendanceService.cs]: attendanceToPut -> updatedAttendance

File: api/Migrations/20220328124415_Attendance.cs (hunk 4 above)
Comment (BarbaMatei) [reply api/Migrations/20220328124415_Attendance.cs:16]: It is renamed in another migration

**Review by Adrian3110 — APPROVED** [review]

**Pull request merged**